src/libecalc/core/models/compressor/train/variable_speed_compressor_train_common_shaft_multiple_streams_and_pressures.py

```diff
@@ -95,6 +96,54 @@ def __init__(
         # previous time step to recirculate. This will take care of that.
         self.fluid_to_recirculate_in_stage_when_inlet_rate_is_zero = [None] * len(self.stages)
 
+    def evaluate_streams(
```

Comment:
It seems that this is not in use yet, just making it ready? Is that so? Possible to use it, and e.g. enable it with a feature flag? Or is it not ready?

Reply:
Used in v2. I'm overwriting the parent method so there is no change where it is used.